src/SonarQube.Client.Tests/Requests/DefaultConfiguration_Configure.cs

```diff
@@ -90,7 +91,7 @@ public void ConfigureSonarCloud_Writes_Debug_Messages()
                     "Registered SonarQube.Client.Api.V5_00.GetSourceCodeRequest",
                     "Registered SonarQube.Client.Api.V5_10.GetLanguagesRequest",
                     "Registered SonarQube.Client.Api.V5_40.GetModulesRequest",
-                    "Registered SonarQube.Client.Api.V5_50.GetRulesRequest",
+                    "Registered SonarQube.Client.Api.V9_6.GetRulesWithEducationPrinciplesRequest",
```

Comment:
Only 9.6 here? what about 9.5?

Reply:
Because SC always uses the latest version.